### PR TITLE
Bug 1135821 - marionette-device-host 1.0.0

### PR DESCRIPTION
**Pull request by gaye**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,125 +1,27 @@
...
+  arguments: {
+    '--port': {
+      defaultValue: 2828,
+      dest: 'port',
+      help: 'Port to use for adb forwarding.',
+      type: 'int'
+    },
+
+    '--restart': {
+      defaultValue: 1,
```
Comment (lightsofapollo) [inline index.js]: you could default these to the env vars if you care about backwards compat

**Comment by lightsofapollo** [comment]
aside from one optional comment above lgtm

File: index.js (hunk 1 above)
Comment (lightsofapollo) [inline index.js]: ^

File: host.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+  args.unshift(cmd);
+
+  var proc = spawn('adb', args);
+  proc.stdout.on('data', ondata);
+  proc.stderr.on('data', ondata);
+
+  return new Promise(function(accept, reject) {
+    proc.on('error', function(error) {
+      proc.removeListener('close', accept);
```
Comment (eliperelman) [inline host.js]: `accept` isn't bound to `close` below, so this won't really do anything.

Comment (gaye) [inline host.js]: Oops! Will fix

**Pull request merged**